frontend/lib/app.dart

```diff
@@ -54,6 +54,13 @@ class App extends StatelessWidget {
 
     final String initialRoute = settings.firstStart ? introRouteName : homeRouteName;
 
+    // Load default language from settings
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      if (settings.language != null) {
+        LocaleSettings.setLocaleRaw(settings.language!);
+      }
+    });
+
```

Comment:
Can we move this to `main.dart` to have everything language/locale related together and not have logic with possible side effects in two places?

Also, why do we need the `addPostFrameCallback`?

Reply:
no because we don't have the context of the `SettingsProvider` here
Because its a lifecycle hook that waits until the widget is rendered to not interfere

Reply:
Thanks for explaining, guessed something like that.